packages/analytics-js-common/src/types/ApplicationState.ts

```diff
@@ -119,10 +120,21 @@ export type SessionState = {
 
 export type SourceConfigState = Signal<Source | undefined>;
 
+export type StorageEntry = {
+  storage: StorageType;
+  key: string;
+};
+export type UserSessionEntries = {
```

Comment:
Let's call it `StorageEntries` as we'll include other items like auth token (DMT) etc. later.